blocks/columns/columns.css

```
@@ -100,6 +100,18 @@ main .columns-container h4 {
   margin-bottom: 0;
 }
 
+.columns.content-band.icon-list.add-bottom-space > div {
+  margin-bottom: var(--spacer-layout-04);
+}
```

bottom spacing should exist for each row, meaning that there's no need to have a class for just applying the bottom spacing.

```suggestion
.columns.content-band.icon-list > div {
  margin-bottom: var(--spacer-layout-04);
}
```